tests/output/cs/10090-string_multi.cs

```diff
@@ -7,7 +7,7 @@ public void F()
       var y = @"
 abc" + "def";
       var z = "" +
-@"
+              @"
```

Comment:
Doesn't this sort of contradict what the test in indent-multistring-coulmn1.cs says?

I would not expect
```cs
var z = "" +
@"
";
```
to be formatted to this
```cs
var z = "" +
            @"
";
```

Reply:
Hi @hschlichter 

Now we have added an option and not enabled the same. so you are getting those diffs.
last time you have merged (mistakenly due to some diff issue) the valid and invalid diffs as a result now those valid also reverted and looks contradict)

if you merge this PR all valid and invalid will be reverted and then later we can create new PR with enabling option in our config then we can see all the valid diffs

let me know if you have any question (i should have explained this before sorry for that :) )

Reply:
@hschlichter this one we discussed int the call and agreed to accept the PR

once this is closed we will create new PR enabling option

Reply:
would actually prefer to have the multiline string on column 1

Reply:
@mihaipopescu once this PR is merged then i can create one proper PR which gives all proper and expected